test/e2e/BUILD

```diff
@@ -244,7 +244,7 @@ genrule(
     testonly = 1,
     srcs = [":go_default_test"],
     outs = ["e2e.test"],
-    cmd = "set -x; srcs=($(SRCS)); cp $$(dirname $${srcs[0]})/go_default_test $@;",
```

Comment:
while you're here, can you also add `output_to_bindir = 1` to this rule? otherwise `e2e.test` ends up in `bazel-genfiles/`.